### PR TITLE
Discontinues model mommy

### PR DESCRIPTION
**Pull request by berinhard**
This PR adds:

- Update the README with links to model bakery;
- Update the docs' index with links to model bakery;
- Add deprecation warning to `mommy/__init__.py`;
- Prepare the new 2.0.0 release;

### CODE REVIEW

File: Changelog.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Development
 -----------
+
+2.0.0
+-----
+- Discontinue the lib and migrate to model_bakery
```
Comment (amureki) [inline Changelog.txt]: Maybe link to repo or pypi would be better.


```suggestion
- Discontinue the lib and migrate to https://github.com/model-bakers/model_bakery
```

File: README.rst [hunk 2]
```diff
@@ -13,18 +10,9 @@ With a simple and powerful API you can create many objects with a single line of
...
-=======
-
-.. code-block:: console
-
-    pip install model_mommy
-
-
-Usage and Info
-==============
+**IMPORTANT:** model_mommy is no longer maintained and was replaced by [model_bakery](https://pypi.org/project/model-bakery/). Please, consider [migrating your project](https://model-bakery.readthedocs.io/en/latest/migrating_from_mommy.html) to use the new lib.
```
Comment (amureki) [inline README.rst]: In restructured format (.rst) links does not work like this. :( Markdown FTW. :D

**Review by amureki — COMMENTED** [review]
So far looks fine to me. Small comments regarding the formatting.

File: docs/source/conf.py [hunk 3]
```diff
@@ -56,9 +56,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.2.1'
+version = '2.0'
```
Comment (anapaulagomes) [inline docs/source/conf.py]:
```suggestion
version = '2.0.0'
```

**Review by anapaulagomes — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (berinhard) [reply README.rst]: My bad, I forgot the readme wasn't markdown. I'll fix the link here :+1:

File: docs/source/conf.py (hunk 3 above)
Comment (amureki) [reply docs/source/conf.py]: @anapaulagomes @berinhard but according to comment that stands for short version:
```
# The short X.Y version.
```

**Pull request merged**